tldw_Server_API/app/api/v1/endpoints/writing_manuscripts.py

```
@@ -819,3 +879,1366 @@ async def delete_scene(
+                result = {"error": f"Unknown analysis type: {analysis_type}"}
+                score = None
+
+            aid = helper.create_analysis(
+                scene["project_id"], "scene", scene_id, analysis_type,
+                result, score=score, provider=provider_override, model=model_override,
+            )
+            analysis = helper.get_analysis(aid)
+            if analysis:
+                results.append(ManuscriptAnalysisResponse(**analysis))
+
+        return results
```

Ruff (0.15.9)

[warning] Consider moving this statement to an `else` block

(TRY300)